gcloud/bigtable/table.py

```diff
@@ -145,3 +147,41 @@ def delete(self):
         client = self._cluster._client
         # We expect a `._generated.empty_pb2.Empty`
         client._table_stub.DeleteTable(request_pb, client.timeout_seconds)
+
+    def sample_row_keys(self):
+        """Read a sample of row keys in the table.
+
+        The returned row keys will delimit contiguous sections of the table of
+        approximately equal size, which can be used to break up the data for
+        distributed tasks like mapreduces.
```

Comment:
I asked Solomon, Doug or Phanee to comment -- I was told (long ago) that HBase (Map/Reduce) does something with this to partition things up.